Lib9c/Model/Stat/CharacterStats.cs

```diff
@@ -326,15 +307,12 @@ public CharacterStats SetBuffs(IEnumerable<Buff.StatBuff> value, bool updateImme
             {
                 var statType = group.Key;
                 var sum = group.Sum(g => g.Value);
-                if (sum > 0L)
-                {
-                    _collectionStatModifiers.Add(new StatModifier(statType, StatModifier.OperationType.Percentage, sum));
-                }
+                _collectionStatModifiers.Add(new StatModifier(statType, StatModifier.OperationType.Percentage, sum));
             }
 
             if (updateImmediate)
             {
-                UpdateEquipmentStats();
+                UpdateCollectionStats();
```

Comment:
Is it OK to replace from equipment to collection? Than where do we update stats from equitments?